gobblin-kubernetes/gobblin-service/basic-cluster/ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: test-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+spec:
+  rules:
+    - host: gaas-test.info
```

Comment:
Is this needed?

Reply:
I would say it's not needed for a minimal ingress as the URL mapping is done on my local machine's side. I can remove this